WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/BoldableTextView.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+
+/// A text view that takes in a string whose substrings within two asterisks are bolded.
+///
+/// For example, initializing with a string "*Never* will I *stop learning*." will bold the substrings "Never" and "stop learning".
```

Comment:
Nitpick: can change * to ** in the comment

Reply:
great catch 😬 updated in: https://github.com/woocommerce/woocommerce-ios/pull/3285/commits/83595186048b3262514e558b9ab386af72af1a61